src/khoj/routers/auth.py

```diff
@@ -117,9 +119,9 @@ async def auth(request: Request):
                 metadata={"user_id": str(khoj_user.uuid)},
             )
             logger.log(logging.INFO, f"New User Created: {khoj_user.uuid}")
-            RedirectResponse(url="/?status=welcome")
+            return RedirectResponse(url=f"{next_url}", status_code=HTTP_302_FOUND)
```

Comment:
Should we still add the `status=welcome` query param here just to use when we add a first-run experience (FRE)?

```suggestion
            return RedirectResponse(url=f"{next_url}/status=welcome", status_code=HTTP_302_FOUND)
```

Reply:
Yes, let's do that once we add the FRE. Confusing for developers otherwise if we have it in there before it's implemented and not being used.